spec/index.html

```diff
@@ -297,10 +297,12 @@ <h3>Terms defined by this specification</h3>
         the algorithm.</dd>
       <dt><dfn>normalized dataset</dfn></dt>
       <dd>The immutable, abstract <a>RDF dataset</a> and a set of normalized
-        <a>blank node identifiers</a> that are produced as output by the algorithm.
-        A <a>normalized dataset</a> is a restriction on an <a>RDF dataset</a>
-        where all nodes are labeled, and <a>blank nodes</a> are labeled with canonical <a>blank node identifiers</a>
-        consistent with running this algorithm on a base <a>RDF dataset</a>.
+        <a>blank node identifiers</a> that are produced as consistent with this algorithm.
+        A <a>normalized dataset</a> is the combination of an <a>RDF dataset</a>
+        and an <a>ordered map</a> where [=map/keys=]
+        are canonical <a>blank node identifiers</a> generated by this
```

Comment:
The choice to have keys be identifiers and values be blank nodes may be backwards, but it makes sense when considering that it is ordered. Alternatively, the key/value could be swapped, and it could be described as being ordered on values, rather than keys. In any case, it's a 1:1 mapping, so it can be reversed easily enough in code.